radix-common/src/data/manifest/model/manifest_address.rs

```diff
@@ -18,9 +18,30 @@ pub enum ManifestAddress {
     /// TODO: prevent direct construction, as in `NonFungibleLocalId`
     Static(NodeId),
     /// Named address, global only at the moment.
-    Named(u32),
+    Named(ManifestNamedAddress),
```

Comment:
Minor: rather than "named", I think `Ref` would be a better name

Reply:
Hmm, I can see this. But then arguably also `ManifestBucket` => `ManifestBucketRef` etc. So I'm not fully sold.

Happy to do some more brainstorming though :). This also applies to `ManifestNamedIntent`